### DISCUSSION TITLE
APU not starting

### DISCUSSION THREAD

**Discussion by andreda** [flybywiresim/aircraft#2663] [Help] [answered]
Just downloaded the stable version, after the game update.... and after pressing APU Master + Start..

APU doesn't work....

anyone experiencing the same?

**Comment by TRock2020** [comment]
Same here! APU is not working. Uninstalled the FlyByWire-A320 - the stable version is working.

**Comment by andreda** [comment]
I am using the stable version, and the APU is not working to me. Can you please describe the steps that you performed? Just to make sure that I'm not missing anything

**Reply by TRock2020** [reply]
ok. just be sure the a320 is cold. verify that the engine is off (Masterswitches 1 and 2) and the startswitch is in normal. Wipers off, battery on - both. APU Master on - APU starter on. Thats all.  Screens will illuminate and so on... Testet 10 min ago. XD

**Comment by andreda** [comment]
hmm... I uninstalled the mod, and now the default plane shows the APU START as UNAVAILABLE...

Update: It seems my default MSFS version is bugged. Even CTRL + E cranks up the engine, but it seems that it doesn't allow fuel flow..  so it ends up shutting down

**Reply by ruimirada** [reply]
I found out that my apu only works if I CTRL + E to start the engines, the apu starts, and then I turn off the engines and the apu stays on for a while and then turns off, I think it's a bug

**Reply by andreda** [reply]
My APU Start stays ON, it never turns into Available (Green).


So the question is... can anyone start the A320 after the latest MSFS Patch?

**Reply by Mike777-200ER** [reply]
My cfbw version has a working APU.  I downloaded it yesterday evening.  Try removing other aircraft -- maybe even liveries -- from the Community folder.  I have noticed that other addons can cause problems.

**Comment by andreda** [comment]
![image](https://user-images.githubusercontent.com/19509499/103037794-af83c400-453a-11eb-90a3-0949b4e9d06e.png)
 Valves are open.. but no fuel hits the engines...

**Comment by andreda** [comment]
Update: I've removed the A320 from in the content manager, re-downloaded, then it seems that it is now normal (DEFAULT). Will install the MOD now.

Update 2: Installed the A320 mod... and again, the APU doesn't work. Am I the only one having this issue with the stable version?

**Reply by Mike777-200ER** [reply]
Sorry, Andreda, but I am a little confused.  Since only the Stable Release version has a version number I suggest the following:  Refer to the Stable Release version 0.5.1, the developers version (add the time and date you downloaded it), and the Custom FBW (or cfbw) version with time and date you downloaded.  I keep careful track of when I downloaded developer versions and cfbw versions as these are continually changing.

As I understand what you did, you downloaded Stable Release version 0.5.1, it worked, and now you are trying out the most recent developers (unstable) version.  

Again, my cfbw version downloaded yesterday,12-22-20 at about 23:25Z has a working APU and flies well, with some known bugs.

**Reply by andreda** [reply]
Thanks MIke,

I didn't download the developers version.. I was using the stock A320.. without any mods... I got the stable one...maybe I'm not expressing myself correctly here.... I am using the installer.. not downloading directly.

![image](https://user-images.githubusercontent.com/19509499/103043322-810de500-454a-11eb-955d-b5f7766723dd.png)

**Comment by andreda** [comment]
I am about to throw the towel :)

Downloaded the CFBW, deleted the current MOD, extracted the newest CFBW... and it problem remains.... it never goes available (green).. and there's no movement on the ECAM.

![image](https://user-images.githubusercontent.com/19509499/103045198-8ae71680-4551-11eb-937a-58c88fb28046.png)

![image](https://user-images.githubusercontent.com/19509499/103045224-aa7e3f00-4551-11eb-8c7a-aa0d4a7e799f.png)

**Reply by HiggsBoson777** [reply]
I have the exact same problem despite following what is provided. Not sure what to do anymore..

**Reply by andreda** [reply]
Yeah, I ended up removing the FBW mod... back to A320 Stock :(

**Reply by HiggsBoson777** [reply]
The only way I found to start the APU is to map a key for APU MASTER and hold the button. Unfortunately, the APU returns to zero when letting go of the button.

**Reply by andreda** [reply]
I've had to delete the A320 folder, and redownload it just to restore to its original state

**Comment by Mike777-200ER** [comment]
I'm not sure what the problem is.  I am guessing that either you have interference from some other addon in the community folder, or somehow you are installing an old version.    I am using the CFBW version.  I wish the A32NX-FBW people would put a version file in all of their downloads.   The two "unstable" versions don't have version numbers, but a file with the date and time it was made available to download would be helpful.

**Reply by ruimirada** [reply]
I finally discovered my problem with the APU, Uwa Light Mod Pack is causing a bug I removed the folder from the comunity folder and the APU is working

**Reply by HiggsBoson777** [reply]
RESOLVED - Dude! You are absolutely right, I removed the UWA Light Mod Pack and boom, the APU works great! Thanks for sharing!

**Reply by andreda** [reply]
Wow. How the heck did you find this out?!?! I have this Light Pack mod, and I just removed... re-downloading FBW to test... Fingers crossed!

Update: Problem solved. :)

**Reply by ruimirada** [reply]
> Wow. How the heck did you find this out?!?! I have this Light Pack mod, and I just removed... re-downloading FBW to test... Fingers crossed!

I found out by trial and error, I kept removing it until I discovered what was doing it,test it and say something

**Reply by ruimirada** [reply]
> Wow. Como diabos você descobriu isso?!?! Eu tenho este light pack mod, e eu acabei de remover ... re-baixar FBW para testar... Dedos cruzados!
> 
> Atualização: Problema resolvido. :)

great

**Reply by Mike777-200ER** [reply]
I recommend "MSFS Addons Linker," at:
https://flightsim.to/file/1572/msfs-addons-linker

It makes it very easy to install and uninstall addons in the Community folder.  Be sure to set Run As Administrator for the program.  

You put your addons in a different folder, where you can organize them.  You can have, for example, an Aircraft folder and even have subfolders, like A32NX, 787 liveries, Scenery>Airports, Scenery>Cities, or whatever categories you want.  This program makes it very easy to troubleshoot problems by adding and removing some or all other addons just by checking and unchecking boxes.

**Reply by andreda** [reply]
Thanks for the suggestion Mike! I'll try that

**Comment by DarkOfNova** [comment, marked as answer]
![Screenshot_3](https://user-images.githubusercontent.com/70452276/105379416-b585c700-5c04-11eb-800f-f09808a265c3.png)
This apparently did it so case closed 😄
